docs/changelog.json

```diff
@@ -2,6 +2,161 @@
   "url": "https://github.com/Shopify/hydrogen/pulls?q=is%3Apr+is%3Aclosed+%5Bci%5D+release+in%3Atitle+is%3Amerged",
   "version": "1",
   "releases": [
+    {
+      "title": "Optimistic variant, Remix upgrade, new session commit pattern, CSS Options back in CLI, Vite bundle analyzer",
+      "version": "2024.7.1",
+      "hash": "b4884b6232383cbf7d64547f48540d0c2fcf56fe",
+      "pr": "https://github.com/Shopify/hydrogen/pull/2236",
+      "commit": "https://github.com/Shopify/hydrogen/commit/b4884b6232383cbf7d64547f48540d0c2fcf56fe",
+      "dependencies": {
+        "@remix-run/react": "^2.10.1",
+        "@shopify/hydrogen": "2024.7.1",
+        "@shopify/remix-oxygen": "^2.0.5"
+      },
+      "devDependencies": {
+        "@remix-run/dev": "^2.10.1",
+        "typescript": "^5.2.2",
+        "@shopify/cli": "^3.64.0",
+        "@shopify/oxygen-workers-types": "^4.1.2",
+        "@remix-run/server-runtime": "^2.10.1"
```

Comment:
`@remix-run/server-runtime` should be moved to `dependencies`.

I'm still somewhat concerned about bumping Remix like this. If they have any other Remix package (like `@remix-run/eslint-config` in our template) it would run out of sync because we don't include everything here 🤔